flexget/plugins/input/anilist.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from flexget.utils.requests import RequestException
+
+log = logging.getLogger('anilist')
+
+LIST_STATUS = ['current', 'planning', 'completed', 'dropped', 'paused', 'repeating']
+
+RELEASE_STATUS = ['finished', 'releasing', 'not_yet_released', 'cancelled', 'all']
+
+ANIME_FORMAT = ['tv', 'tv_short', 'movie', 'special', 'ova', 'ona', 'all']
+
+TRAILER_SOURCE = {
+    'youtube': 'https://www.youtube.com/embed/',
+    'dailymotion': 'https://www.dailymotion.com/embed/video/'
+    }
+class AniList(object):
+    """" Creates entries for series and movies from your AniList list
+
+    Syntax:
+    anilist:
+      username: <value>
+      status:
+        - <current|planning|completed|dropped|paused|repeating>
+        - <current|planning|completed|dropped|paused|repeating>
+        ...
+      release_status:
+        - <all|finished|releasing|not_yet_released|cancelled>
+        - <finished|releasing|not_yet_released|cancelled>
+        ...
+      format:
+        - <all|tv|tv_short|movie|special|ova|ona>
+        - <tv|tv_short|movie|special|ova|ona>
+        ...
+    """
+
+    schema = {
+        'oneOf': [
+            {'type': 'string'},
+            {
+                'type': 'object',
+                'properties': {
+                    'username': {'type': 'string'},
+                    'status': one_or_more(
+                        {'type': 'string', 'enum': LIST_STATUS}, unique_items=True
+                    ),
+                    'release_status': one_or_more(
+                        {'type': 'string', 'enum': RELEASE_STATUS}, unique_items=True
+                    ),
+                    'format': one_or_more(
+                        {'type': 'string', 'enum': ANIME_FORMAT}, unique_items=True
+                    )
+                },
+                'required': ['username'],
+                'additionalProperties': False,
+            },
+        ]
+    }
+
+    @cached('anilist', persist='2 hours')
+    def on_task_input(self, task, config):
+        entries = []
```

Comment:
I'd add something here to make sure your config is always a dict, otherwise your below statements could fail in the case of the simple string configuration. (e.g. `anilist: status_in_my_username`)
```python
if isinstance(config, str):
    config = {'username': config}
```